twtr/grails-app/controllers/UrlMappings.groovy

```diff
@@ -23,6 +23,10 @@ class UrlMappings {
             }
         }
 
+        "/search"(controller:'message') {
+            action = [GET: 'search']
+        }
```

Comment:
as an FYI, this mapping actually isn't needed.  the default mapping will handle it.